shaders/shader.frag

```diff
@@ -15,9 +15,10 @@ void main() {
     //phong-shading code in case we need to add it back
     //float ambient = 0.1;
     //float diffuse = clamp(dot(norm, normalize(lightPos - pos)), 0.0, 1.0);
+
```

Comment:
can we rely on git to get back to this code if needed?

Reply:
I Removed it.